src/glyphs.js

```diff
@@ -140,26 +140,35 @@ const updateMessages = (id, path, req) => {
 /* Constant for what the name in the "gauge" field of the messages spreadsheet
 * should be when creating a message attached to a view.
 */
-const viewPath = "Intro"
+const viewPath = "intro"
 
 const importMessages = (line) => {
   const message = line.split(",");
   const viewMessage = (message[1] === viewPath);
 
+
   const newMessage = {"text": message[2], "probability": (viewMessage) ? Number(message[3]):message.splice(3, 5).map(num => Number(num))};
 
   const path = (viewMessage) ? "view" : `view.gauges.${message[1]}`;
 
-  return db.collection.updateOne(
-    {
-      "view.name": message[0].toLowerCase()
-    },
-    {
+  //Not a big fan of how this works but unsure how to handle if it's an "Intro" message
```

Comment:
Yep, we can turn this into one `db.collection.updateOne` by making the `if` statements more specific.  So, what is different in the calls?  The query and what we are setting.  Both are objects, so we can set them outside of the call.  We have two variables in the query object:
`"view.name": message[0].toLowerCase()` -> this always exists because two gauges could have the same name across two views.
`"view.gauges.name": message[1].toLowerCase()` -> this can be conditionally added depending on if the message is a view intro or not.

Now, the set object: this will be set to `newMessage` no matter what, so you need to conditionally set the key.  This is done in the old code as `[`${path}.messages`]`.  Can simply set `path` conditionally up above.  Hope this helps.

Reply:
I guess I'm confused on the difference in mongoDB between using double quotes like `"view.gauges.name"` vs. the array brackets `[${path}.messages]` for keying. They aren't the same right?

Reply:
The array brackets notation is actually a JavaScript thing, and not a MongoDB thing.
https://www.samanthaming.com/tidbits/37-dynamic-property-name-with-es6/
So, the string within array brackets is actually being expanded with the value of the variable `path`.

Does that link clear things up?